modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -43,6 +43,30 @@ For more information, refer to the link:https://argocd-operator.readthedocs.io/e
 
 * With this update, you can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
 
+* With this update, {gitops-title} provides support for the `sidecar` and `init` containers for the Server and Application Controller components. You can configure the `sidecar` and `init` containers by using the `spec.server.sidecarContainers`, `spec.controller.sidecarContainers`, `spec.server.initContainers` and `spec.controller.initContainers` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-5010[GITOPS-5010]
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  server:
+    initContainers:
+    - name: <name of the initContainer> #<1>
+      image: <image name of the initContainer> #<2>
+    server:
```

Comment:
isnt this line wrong?

Reply:
hey @Dhruv-Soni11, yeah that's a typo. You don't need `server` here.

Pasting the correct example for your reference.
```
apiVersion: argoproj.io/v1beta1
kind: ArgoCD
metadata:
  name: example-argocd
spec:
  server:
    initContainers:
    - name: <name of the initContainer> #(1)
      image: <image name of the initContainer> #(2)
    sidecarContainers:
    - name: <name of the sidecarContainer> #(3)
      image: <image name of the sidecarContainer> #(4)
```

Reply:
Fixed.